File: Code_C/Exercicio-8/calcular-fatorial-num-inteiro.cpp
#include <stdio.h>
#include <stdlib.h>
int main(){
	
	int num, x = 2, fat = 1;
	
	printf("==============================================================\n");
	printf("          Encontre o resultado de um num fatorial \n");
	printf("==============================================================\n");
	printf("Digite o valor do numero: ");
	scanf("%d", &num);
	while (x <= num){
	    fat = fat * x;
	    x = x +1;
    }
	printf("O resultado do valor de [%d!] eh: %d\n", num, fat);
	printf("==============================================================\n");
system("PAUSE");
return(0);
}

File: Code_Python/Exercicio-09/calcular-fatorial-num-inteiro.py
#questão 07
print('='*50)
print(" " * 5, "Encontre o resultado de um n° fatorial")
print('='*50)
num = int(input("Digite o valor do numero: "))
fat = 1
x = 2
while x <= num:
    fat = fat * x
    x = x +1
print(f"O resultado do valor de [{num}!] é: {fat}")